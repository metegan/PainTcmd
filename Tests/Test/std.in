LOAD dessin.txt
PL pl 1 2 3 3 3 3
SAVE dessin2.txt
EXIT

