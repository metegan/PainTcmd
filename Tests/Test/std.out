OK
OK
# New object : pl
OK
